src/status.ts

```diff
@@ -0,0 +1,26 @@
+import { database } from './database/db'
+import { LAST_BLOCKS_TABLE_NAME } from './indexer/blocks'
+import { VERSION } from './config'
+import { Event } from './indexer'
+import { getContractKit } from './util/utils'
+import { Request, Response } from 'express'
+
+export function getStatusHandler(startTime: number) {
+  return async (_req: Request, res: Response) => {
```

Comment:
Should we wrap this with an [asyncRoute](https://github.com/valora-inc/in-house-liquidity/blob/main/src/routes/async-route.ts) to avoid unhandled promise rejections?

Reply:
sure